python/meterpreter/meterpreter.py

```diff
@@ -22,14 +22,6 @@
 else:
 	has_windll = hasattr(ctypes, 'windll')
 
-# this MUST be imported for urllib to work on OSX
```

Comment:
@zeroSteiner you originally added this, but I haven't seen it needed on 10.9 -10.11. Do you recall what the precondition or python version was to cause the problem?